layers/+tools/systemd/packages.el

```diff
@@ -40,4 +40,15 @@
               "hd" 'systemd-doc-directives
               "ho" 'systemd-doc-open)))
 
+(defun systemd/init-journalctl-mode ()
+  (use-package journalctl-mode
+    :ensure t
```

Comment:
What's the purpose of having `:ensure t` instead of deferring it like all the other packages?